docs/getting-started/release-notes/v2_11_0.md

```diff
@@ -10,6 +10,18 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
 
 Zowe Version 2.11.0 contains the enhancements that are described in the following topics.
 
+### Zowe Installation and Packaging
+
+- Added `zowe.sysMessages` as a property to the `Zowe YAML` configuration file. It is a new array that allows you to select messages that when found in Zowe’s job log, will be duplicated into the z/OS syslog. ([#93](https://github.com/zowe/launcher/pull/93))
```

Comment:
I don't quite understand the grammar of this sentence. "It is a new array that allows you to select messages that when found in Zowe’s job log, will be duplicated into the z/OS syslog." Is it the array that is duplicated? Perhaps this should be two sentneces.